event/src/test/java/com/facebook/airlift/event/client/FixedDummyEventClass.java

```diff
@@ -15,20 +15,19 @@
  */
 package com.facebook.airlift.event.client;
 
-import org.joda.time.DateTime;
```

Comment:
same as above. Let us keep joda classes and just add ZonedDateTime as an additional var.

Reply:
had to remove because of checkstyle error.

[ERROR] src/test/java/com/facebook/airlift/event/client/FixedDummyEventClass.java:[18,8] (imports) UnusedImports: Unused import - org.joda.time.DateTime.
[ERROR] src/test/java/com/facebook/airlift/event/client/FixedDummyEventClass.java:[19] (imports) ImportOrder: 'java.time.ZonedDateTime' should be separated from previous imports.

Reply:
Unused imports happen when the class is not used. Obviously that should not be the case here.